src/solvers/dgmulti/dg_parabolic.jl

```diff
@@ -147,22 +204,28 @@ function calc_single_boundary_flux!(flux_face_values, u_face_values, t,
   md = mesh.md
```

Comment:
This was from a previous PR - I believe we were considering removing `u_face_values` as an input into `boundary_condition` since it's not used currently. We left it in for generality. 

I can go ahead and remove it.